packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawer.stories.tsx

```diff
@@ -8,32 +8,63 @@ import {
   NavSubItem,
   NavSubItemGroup,
 } from '@fluentui/react-nav-preview';
-import { DrawerBody } from '@fluentui/react-drawer';
-import { makeStyles, shorthands, tokens } from '@fluentui/react-components';
+import { DrawerBody, DrawerProps } from '@fluentui/react-drawer';
+import { Button, Label, Radio, RadioGroup, makeStyles, shorthands, tokens, useId } from '@fluentui/react-components';
 import { Folder20Filled, Folder20Regular, bundleIcon } from '@fluentui/react-icons';
+
 const useStyles = makeStyles({
   root: {
+    ...shorthands.border('2px', 'solid', '#ccc'),
```

Comment:
Should the color be a token? the stroke width should also be a token, I think it's strokeWidthThin(?)